templates/asc-alert-send-notification-email.json

```diff
@@ -0,0 +1,99 @@
+{
+    "id": "/providers/Microsoft.Logic/galleries/public/templates/d27feb15-dc1d-4068-93c5-d9dca32806cd",
+    "type": "Microsoft.Logic/galleries/templates",
+    "name": "d27feb15-dc1d-4068-93c5-d9dca32806cd",
+    "properties": {
+        "author": {
+            "displayName": "Microsoft"
+        },
+        "categoryNames": [
+            "general",
+			"security"
+        ],
+        "description": "Send an email notification with the alert details when a threat is detected",
+        "displayName": "Get a notification email when Security Center detects a threat",
```

Comment:
Make sure there's no text clipping in UI

Reply:
I checked that the texts are not cut in the UI if that is what you mean.
Is there a specific way to check that there is no text clipping in UI?